inc/storefront-template-functions.php

```diff
@@ -366,7 +366,7 @@ function storefront_post_header() {
 				storefront_posted_on();
 			}
 
-			the_title( sprintf( '<h2 class="alpha entry-title"><a href="%s" rel="bookmark">', esc_url( get_permalink() ) ), '</a></h2>' );
+			the_title( sprintf(  wp_kses ( __( '<h2 class="alpha entry-title"><a href="%s" rel="bookmark">', esc_url( get_permalink() ) ), '</a></h2>' ) ) );
```

Comment:
There is no need for `__()`, since it's just pain HTML here, there is no text do translate.